### PR TITLE
Update demo data

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Adding addtional data for demo

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [X] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/libs/efiling-demo-starter/src/test/java/ca/bc/gov/open/jag/efiling/demo/EfilingReviewServiceDemoImplTest.java [hunk 1]
```diff
@@ -101,6 +101,163 @@ public void withEmptyCacheShouldReturnPackage() {
 
     }
 
+    @Test
```
[the remark this answers is not in the document]
Comment (TayGov) [reply src/backend/libs/efiling-demo-starter/src/test/java/ca/bc/gov/open/jag/efiling/demo/EfilingReviewServiceDemoImplTest.java:104]: Test

[the remark this answers is not in the document]
Comment (TayGov) [reply src/backend/libs/efiling-demo-starter/src/test/java/ca/bc/gov/open/jag/efiling/demo/EfilingReviewServiceDemoImplTest.java:104]: Test

File: src/backend/libs/efiling-demo-starter/src/test/java/ca/bc/gov/open/jag/efiling/demo/EfilingReviewServiceDemoImplTest.java [hunk 2]
```diff
@@ -101,6 +101,163 @@ public void withEmptyCacheShouldReturnPackage() {
...
+        Assertions.assertTrue(result.get().getPayments().get(1).getFeeExmpt());
+        Assertions.assertEquals(new BigDecimal(3), result.get().getPayments().get(1).getPaymentCategory());
+        Assertions.assertEquals(BigDecimal.ONE, result.get().getPayments().get(1).getSubmittedAmt());
+        Assertions.assertEquals("Affidavit", result.get().getPayments().get(1).getTransactionDesc());
+
+        Assertions.assertEquals("http://localhost:8080/wherearemypackage", result.get().getPackageLinks().getPackageHistoryUrl());
+
+    }
+
+    @Test
```
[the remark this answers is not in the document]
Comment (TayGov) [reply src/backend/libs/efiling-demo-starter/src/test/java/ca/bc/gov/open/jag/efiling/demo/EfilingReviewServiceDemoImplTest.java:183]: Test

**Review by karthikmsqa — APPROVED** [review]
Awesome! thanks @TayGov

**Pull request merged**